manifests/plugin/apache.pp

```diff
@@ -2,11 +2,10 @@
 class collectd::plugin::apache (
   Enum['present', 'absent'] $ensure        = 'present',
   Boolean $manage_package                  = $collectd::manage_package,
-  Hash $instances                          = { 'localhost' => { 'url' => 'http://localhost/mod_status?auto' } },
+  Hash $instances                          = { 'localhost' => { 'url' => 'http://localhost/mod_status?auto' }},
```

Comment:
Why is this space removed, but not the others?

Reply:
As a side note, I had a similar case where I had a default value for a Hash (but more data and thus multiline), which crashed [class_parameter-check](https://rubygems.org/gems/puppet-lint-class_parameter-check)